pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/openshift/windows-machine-config-operator/version"
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/ssh"
-	"k8s.io/api/core/v1"
+	core "k8s.io/api/core/v1"
```

Comment:
Ideally such changes should go into a separate PR.

Reply:
Alright, I will revert for now and we can have a separate PR at some point to standardize this across the repo.